drizzlepac/hapsequencer.py

```diff
@@ -57,17 +60,23 @@
 # --------------------------------------------------------------------------------------------------------------
 
 
-def correct_hla_classic_ra_dec(orig_hla_classic_sl_name, cattype, log_level):
+def correct_hla_classic_ra_dec(orig_hla_classic_sl_name, hap_imgname, cattype, log_level):
```

Comment:
What is different about this function compared to the one in 'comparison_utils/integrate_read_hla_catalog.py'?  If they are the same (or can be made the same), we should only be maintaining and using a single copy of this code.

Reply:
comparison_utils/integrate_read_hla_catalog.py was originally written to simply speed the testing and integration of Rick's read_hla_catalog.py into hapsequencer.py. The inputs for the version of correct_hla_classic_ra_dec in hapsequencer.py probably changed a bit since introduction. Since everything has been integrated properly, I'm just going to remove comparison_utils/integrate_read_hla_catalog.py.

Reply:
Issue resolved in commit [e68b828](https://github.com/spacetelescope/drizzlepac/pull/602/commits/e68b8282897f6f4433d3297a53e9127cf06133b4)